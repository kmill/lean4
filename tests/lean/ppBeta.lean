#check (fun x => x) Nat

set_option pp.beta true

#check (fun x => x) Nat
