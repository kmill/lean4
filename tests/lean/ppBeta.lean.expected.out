(fun x => x) Nat : Type
Nat : Type
